app/vmui/packages/vmui/src/utils/time.ts

```diff
@@ -151,3 +146,31 @@ export const getRelativeTime = ({ relativeTimeId, defaultDuration, defaultEndInp
     endInput: target ? target.until() : defaultEndInput
   };
 };
+
+export const getUTCByTimezone = (timezone: string) => {
+  const date = dayjs().tz(timezone);
+  return `UTC${date.format("Z")}`;
+};
+
+export const getTimezoneList = (search = "") => {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  const arrTimezone = Intl.supportedValuesOf("timeZone") as string[];
+  const regexp = new RegExp(search, "i");
+
+  return arrTimezone.reduce((acc: {[key: string]: Timezone[]}, region) => {
+    const zone = (region.match(/^(.*?)\//) || [])[1] || "unknown";
+    const utc = getUTCByTimezone(region);
+    const item = {
+      region,
+      utc,
+      search: `${region} ${utc} ${region.replace(/[/_]/gmi, " ")}`
+    };
+
+    if (!search || (search && regexp.test(item.search))) {
+      acc[zone] ? acc[zone].push(item) : acc[zone] = [item];
```

Comment:
it is better to use if else combination instead of the ternary operator with assignment

Reply:
Agreed - the `if ... else ...` code is easier to read and maintain than the trenary operator :)